src/awkward/operations/ak_from_rdataframe.py

```diff
@@ -3,13 +3,24 @@
 import awkward as ak
 
 
-def from_rdataframe(rdf, columns, offsets_type="int64_t", keep_order=False):
+def from_rdataframe(
+    rdf,
+    columns,
+    highlevel=True,
+    behavior=None,
+    offsets_type="int64_t",
+    keep_order=False,
+):
     """
     Args:
         rdf (`ROOT.RDataFrame`): ROOT RDataFrame to convert into an
             Awkward Array.
         columns (str or iterable of str): A column or multiple columns to be
             converted to Awkward Array.
+        highlevel (bool): If True, return an #ak.Array; otherwise, return
+            a low-level #ak.contents.Content subclass.
+        behavior (None or dict): Custom #ak.behavior for the output array, if
+            high-level.
         offsets_type (str): A C++ type of the ListOffsetArray offsets.
         keep_order (bool): If set to `True` the columns with Awkward type will
             keep order after filtering.
```

Comment:
```suggestion
        offsets_type (str): A C++ type of the ListOffsetArray offsets.
        keep_order (bool): If set to `True` the columns with Awkward type will
            keep order after filtering.
        highlevel (bool): If True, return an #ak.Array; otherwise, return
            a low-level #ak.contents.Content subclass.
        behavior (None or dict): Custom #ak.behavior for the output array, if
            high-level.
```